actions-sms.php

```diff
@@ -241,140 +233,135 @@ function freeBikes($number)
 
 function log_sms($sms_uuid, $sender, $receive_time, $sms_text, $ip)
 {
-    global $dbserver,$dbuser,$dbpassword,$dbname;
```

Comment:
Is this ok ? I saw in other files that these credentials are used for the main db too, so I removed this db creation.